### ISSUE TITLE
Minor bug in rv multipanel plot

### ISSUE THREAD

**Issue by zhexingli** [California-Planet-Search/radvel#350]
There's a small bug when displaying the derived mpsini in the rv multipanel plot. See attached.

The derived mpsini is something like 0.1 solar mass but in the plot it displays 30 solar masses for some reasons

[HD173902_radvel_results.pdf](https://github.com/California-Planet-Search/radvel/files/6989049/HD173902_radvel_results.pdf)

A unrelated question.... in the derived_quantiles.csv file, is the unit for 'mpsini' always in Earth unit? What's 'musini' and what's the unit for it? Ty.

**Comment by bjfultn** [comment]
Yes, I believe mpsini is always in Earth units and musini is the mass ratio.

I'll try to track down that other little bug.

**Comment by bjfultn** [comment]
I looked into this for a few min, but I couldn't quickly find the culprit. @sealauren perhaps a student has time to dig into it and search for the bug?

**Comment by bjfultn** [comment]
Fixed in next-release branch: changed `if` to `elif` in `mcmc_plots.py` line 215 for the mpsini unit conversion chain. The second condition was unreachable because both conditions checked the same median value. Will be included in version 1.5.1 (PR #405).
